build/coverage.sh

```diff
@@ -6,28 +6,28 @@ set -e
 
 declare -r ROOT=$(realpath $(dirname $0)/..)
 declare -r TEST=$ROOT/src/NodaTime.Test
-declare -r DOTCOVER_VERSION=2019.3.4
-declare -r REPORTGENERATOR_VERSION=4.0.12
+declare -r DOTCOVER_VERSION=2021.1.3
+declare -r REPORTGENERATOR_VERSION=4.8.11
 
-nuget.exe install -Verbosity quiet -OutputDirectory $ROOT/packages -Version $DOTCOVER_VERSION JetBrains.dotCover.CommandLineTools
+export REPORTGENERATOR_VERSION
+export DOTCOVER_VERSION
+export DOTCOVER_PACKAGE_SUFFIX
+dotnet restore $ROOT/build/Coverage.proj --packages $ROOT/packages
 
-declare -r DOTCOVER=$ROOT/packages/JetBrains.dotCover.CommandLineTools.$DOTCOVER_VERSION/tools/dotCover.exe
+declare -r DOTCOVER_DIR=$ROOT/packages/jetbrains.dotcover.commandlinetools${DOTCOVER_PACKAGE_SUFFIX}/$DOTCOVER_VERSION/tools
 
 rm -rf $ROOT/coverage
 mkdir $ROOT/coverage
 
 # Run the tests under dotCover
-(cd $TEST; 
- $DOTCOVER cover coverageparams.xml -ReturnTargetExitCode)
-
-$DOTCOVER report --Source=$ROOT/coverage/NodaTime.dvcr --Output=$ROOT/coverage/coverage.xml --ReportType=DetailedXML ""
+(cd $DOTCOVER_DIR; 
```

Comment:
We've lost the -ReturnTargetExitCode parameter here - was that deliberate because the behavior has changed, or accidental?

Reply:
I thought at one point that I was getting a "not used parameter" warning from dotCover for ReturnTargetExitCode, but I just checked and that is not the case. I added it back.